package.json

```diff
@@ -115,7 +115,8 @@
     "tiny-lr": "^1.0.3",
     "tree-sync": "^1.1.4",
     "walk-sync": "^0.3.0",
-    "yam": "0.0.22"
+    "yam": "0.0.22",
+    "yarn": "^0.17.8"
```

Comment:
@nathanhammond can we move it to `devDependencies`, ship, then iterate on wiring it into `ember new/addon/init`?